horqrux/matrices.py

```diff
@@ -23,11 +20,11 @@
         [0, 0.5 * (1 - 1j), 0.5 * (1 + 1j), 0],
         [0, 0, 0, 1],
     ],
-    dtype=jnp.complex128,
+    dtype=jnp.complex64,
```

Comment:
hm isn't default in PyQ `complex128` ?

Reply:
Yes, not sure why was changed by ds

Reply:
@RolandMacDoland a `default_complex_dtype` should do the trick for allowing complex128 if the environment variable is set